src/graphql/search/data/index.ts

```diff
@@ -1,18 +1,22 @@
-import { DEFAULT_FROM, DEFAULT_LIMIT } from '../../../config'
 import { DataSearchResult, QueryDataSearchArgs } from '../../../generated/graphql'
 import { combineTypeResults } from './normalize'
-import { DATA_SEARCH_ENDPOINTS } from './config'
+import { DATA_SEARCH_ENDPOINTS, DATA_SEARCH_LIMIT, DATA_SEARCH_MAX_RESULTS } from './config'
 import getFilters from './filters'
 import { Context } from '../../config'
+import getPageInfo from '../../utils/getPageInfo'
 
 const index = async (
   _: any,
   { q: searchTerm, input }: QueryDataSearchArgs,
   context: Context,
 ): Promise<DataSearchResult> => {
-  const { limit, from, filters } = input || {}
+  let { page } = input || {}
+  const { filters } = input || {}
   const { loaders } = context
 
+  // Get the page from the input, otherwise use the default
+  page = page || 1
```

Comment:
`??`

Reply:
Same: In this case the `||` is better, as the page must not be zero. Using the `&&` will result in users being able to navigate to page zero